frontend.md

```diff
@@ -0,0 +1,192 @@
+
+<a name="using-php"></a>
+## Using PHP
+
+<a name="php-and-blade"></a>
+### PHP & Blade
+
+In the past, most PHP applications rendered HTML to the browser using simple HTML templates interspersed with PHP `echo` statements to render data that was retrieved from a data store during the request:
+
+```blade
+<div>
+    <?php foreach ($users as $user): ?>
+        Hello, <?php echo $user->name; ?> <br />
+    <?php endforeach; ?>
+</div>
+```
+
+In Laravel, this approach to rendering HTML can still be achieved using [views](/docs/{{version}}/views) and [Blade](/docs/{{version}}/blade). Blade is an extremely light-weight templating language that provides convenient, short syntax for display data, iterating over data, and more:
+
+```blade
+<div>
+    @foreach ($users as $user)
+        Hello, {{ $user->name }} <br />
+    @endforeach
+</div>
+```
+
+When building applications in this fashion, form submissions and other page interactions typically receive an entirely new HTML document from the server and the entire page is re-rendered. Even today, many applications may be perfectly suited to having their frontends constructed in this way using simple Blade templates.
+
+<a name="growing-expectations"></a>
+#### Growing Expectations
+
+However, as user expectations for web applications have matured, many developers have found the need to build more dynamic frontends with interactions that feel more polished. In light of this, some developers choose to begin building their application's frontend using JavaScript frameworks such as Vue and React.
+
+Others, preferring to stick with the backend language they are comfortable with, have developed solutions that allow the construction of modern web application UIs while still primarily utilizing their backend language of choice. In other ecosystems, this has spurred the creation of libraries such as [Turbo](https://turbo.hotwired.dev/) and [Hotwire](https://hotwired.dev/).
+
+Within the Laravel ecosystem, the need to create modern, dynamic frontends by primarily using PHP has led to the creation of [Laravel Livewire](https://laravel-livewire.com) and [Alpine.js](https://alpinejs.dev/).
+
+<a name="livewire"></a>
+### Livewire
+
+[Laravel Livewire](https://laravel-livewire.com) is a framework for building Laravel powered frontends that feel dynamic, modern, and alive just like frontends built with modern JavaScript frameworks like Vue and React.
+
+When using Livewire, you will create Livewire "components" that render a discrete portion of your UI and exposes methods and data that can be invoked and interacted with from your application's frontend. For example, a simple "Counter" component might look like the following:
+
+```php
+use Livewire\Component;
```

Comment:
For consistency with the Inertia example below, do you think we should add this?

```suggestion
<?php

namespace App\Http\Livewire;

use Livewire\Component;
```